### ISSUE TITLE
Understanding LR

### ISSUE THREAD

**Issue by dariogonle** [ultralytics/yolov5#3135]
## ❔Question
Hi,

 I notice that the learinig rate changes among epochs. Could someone explain me how? In hyp.yaml there is a lr and a lf, what means that lr=0.01 and lf=0.2?

I see that there are 3 different learning rates what does they mean?

![image](https://user-images.githubusercontent.com/80393891/117967755-b95d6680-b325-11eb-82d3-87fe8bd79df3.png)

**Comment by glenn-jocher** [comment]
@dariogonle each of 3 parameter groups in YOLOv5 has a separate LR policy and scheduler. The 3 groups are defined in train.py here:
https://github.com/ultralytics/yolov5/blob/abfcf9eb79877971acd238cafe6149711c5056ad/train.py#L109-L133

**Comment by glenn-jocher** [comment]
@dariogonle the lr and lr0 comments explain their usage. They are applied in the LR scheduler here:
https://github.com/ultralytics/yolov5/blob/abfcf9eb79877971acd238cafe6149711c5056ad/train.py#L134-L140
